### PR TITLE
fix: add luit test

### PR DESCRIPTION
**Pull request by alswl**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by alswl** [comment]
DO NOT merge now, the bin test fail in my mac. Maybe it should build in users mac instead of bottles.

File: Formula/luit.rb [hunk 1]
```diff
@@ -13,11 +13,13 @@ class Luit < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--prefix=#{prefix}", "--with-encodings-dir=auto"
```
Comment (fxcoudert) [inline Formula/luit.rb]: Why does this change, and why is it needed?

File: Formula/luit.rb [hunk 2]
```diff
@@ -13,11 +13,13 @@ class Luit < Formula
...
     system "make", "install"
   end
 
   test do
-    system "#{bin}/luit", "-list"
+    require "pty"
+    PTY.spawn(bin/"luit", "-encoding", "GBK", "ls") do |r, _w, _pid|
```
Comment (fxcoudert) [inline Formula/luit.rb]: What is the test trying to do?

File: Formula/luit.rb (hunk 1 above)
Comment (alswl) [reply Formula/luit.rb]: The `encodings-dir` is auto detected by `configure` default. If build on Linux server, the `encodings-dir` maybe `/usr/share/fonts/X11`. But this path does not exists on macOS.
It's strange behavior here, if I build on my mac, the build binary works well, but if it build from Jenkins builder, the build binary abort with message "cannot open /usr/share/fonts/X11" .

File: Formula/luit.rb (hunk 2 above)
Comment (alswl) [reply Formula/luit.rb]: Run binary with `-encoding GBK` will call `encodings-dir`, but this require `tcgetattr` in `pty`, I found solution here https://github.com/Homebrew/homebrew-core/search?q=tcgetattr&type=Issues

**Pull request closed without merging**